flink-libraries/flink-table/src/test/java/org/apache/flink/api/java/table/test/StringExpressionsITCase.java

```diff
@@ -40,46 +40,6 @@ public StringExpressionsITCase(TestExecutionMode mode) {
 		super(mode);
 	}
 
-	@Test(expected = CodeGenException.class)
```

Comment:
Why did you remove these tests? The `ScalarFunctionsTest` does not test the feature end-to-end, right?